modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -116,9 +116,25 @@ if (journalContentDisplayContext.isShowArticle()) {
 						</div>
 					</c:when>
 					<c:when test="<%= article.isScheduled() && !journalContentDisplayContext.isPreview() %>">
-						<div class="alert alert-warning">
-							<liferay-ui:message arguments="<%= new Object[] {HtmlUtil.escape(article.getTitle(locale)), dateFormatDateTime.format(article.getDisplayDate())} %>" key="x-is-scheduled-and-will-be-displayed-on-x" />
-						</div>
+
+						<%
+						AssetRenderer<JournalArticle> assetRenderer = assetRendererFactory.getAssetRenderer(article.getResourcePrimKey());
+						%>
+
+						<c:choose>
+							<c:when test="<%= assetRenderer.hasEditPermission(permissionChecker) %>">
+								<div class="alert alert-warning">
+									<a href="<%= assetRenderer.getURLEdit(liferayPortletRequest, liferayPortletResponse, WindowState.NORMAL, currentURLObj) %>">
+										<liferay-ui:message arguments="<%= HtmlUtil.escape(article.getTitle(locale)) %>" key="x-is-not-approved" />
+									</a>
+								</div>
+							</c:when>
+							<c:otherwise>
+								<div class="alert alert-warning">
+									<liferay-ui:message arguments="<%= new Object[] {HtmlUtil.escape(article.getTitle(locale)), dateFormatDateTime.format(article.getDisplayDate())} %>" key="x-is-scheduled-and-will-be-displayed-on-x" />
+								</div>
+							</c:otherwise>
+						</c:choose>
```

Comment:
Try to refactor this with the case (!article.isApproved() && !journalContentDisplayContext.isPreview())